check-diff.sh

```diff
@@ -494,12 +494,18 @@ function run_phpunit_local {
 				cd - > /dev/null
 			fi
 
-			if [ ! -z "$ABSOLUTE_VAGRANT_PATH" ]; then
+            if [ ! -z "$ABSOLUTE_VAGRANT_PATH" ]; then
 				echo "Running phpunit in Vagrant"
-				vagrant ssh -c "cd $ABSOLUTE_VAGRANT_PATH && phpunit -c $PHPUNIT_CONFIG"
+				if [ -n "$PHPUNIT_CONFIG" ]; then
+				    vagrant ssh -c "cd $ABSOLUTE_VAGRANT_PATH && phpunit -c $( echo "$PHPUNIT_CONFIG"; )"
+				else
+				    vagrant ssh -c "cd $ABSOLUTE_VAGRANT_PATH && find . -iname phpunit.xml.dist -execdir phpunit \;"
```

Comment:
Here's one possibility for a refactor:

```bash
vagrant ssh -c "
cd $ABSOLUTE_VAGRANT_PATH;
export DIFF_BASE=$DIFF_BASE;
export DIFF_HEAD=$DIFF_HEAD;
export DEV_LIB_ONLY=phpunit
.git/hooks/pre-commit
"
```

Note the `DEV_LIB_ONLY` is not currently recognized, but it could be a way to do the inverse of `DEV_LIB_SKIP`,

Just an idea.